Branch/BranchQRCode.m

```diff
@@ -0,0 +1,211 @@
+//
+//  BranchQRCode.m
+//  BranchMacOS
+//
+//  Created by Nipun Singh on 5/23/22.
+//  Copyright © 2022 Branch, Inc. All rights reserved.
+//
+
+#import <LinkPresentation/LPLinkMetadata.h>
+#import "BranchQRCode.h"
+#import "Branch.h"
+#import "BNCQRCodeCache.h"
+
+@implementation BranchQRCode
+
+NSString *buoTitle;
+CIImage *qrCodeImage;
+
+- (void) setMargin:(NSNumber *)margin {
```

Comment:
There is no default value of margin and width ( no init function). If user doesn't set margin and width, what will be behavior ? QR code will generate code automatically with some server side default values ?

Reply:
Yeah, there are server-side defaults so all of the QR code settings are optional.